### DISCUSSION TITLE
Mobile friendly keyboard layout

### DISCUSSION THREAD

**Discussion by bootkernel** [StereoKit/StereoKit#1109] [General]
Any estimates on when this mobile friendly layout will be shipped? https://github.com/StereoKit/StereoKit/blob/ca2be7d45f4f4388e8df7542e9a0313bcc45946e/StereoKitC/tools/virtual_keyboard.cpp#L180

I was going through the code but wasn't sure what this touch functionality meant.

**Comment by maluoi** [comment]
The touch functionality is basically just the ability to touch the UI.Input element to place the cursor position! The current keyboard layout contains arrow keys to get around this, but the "mobile" layout there does not. 

You can call these functions with custom layouts from the top level already in v0.4 preview builds! So if you want to use this layout, or use the layout as a base for your own keyboard, try it with `platform_keyboard_set_layout`, or `Platform.KeyboardSetLayout`. See here for how that works:
https://github.com/StereoKit/StereoKit/blob/develop/Examples/StereoKitTest/Demos/DemoTextInput.cs#L65-L70

**Reply by bootkernel** [reply]
Ahh I see, I'm going to try that! Thank you for the pointers!
